devtools-ci-index.yaml

```diff
@@ -276,8 +276,10 @@
     <<: *job_template_defaults
 
 - job-template:
-    name: '{ci_project}-{git_repo}-f8planner-snapshot'
-    image_name: 'MISSING_IMAGE_NAME'
+    name: '{ci_project}-{git_repo}-f8planner'
+    REGISTRY: 'MISSING_REGISTRY'
+    IMAGE_REPO: 'MISSING_IMAGE_REPO'
+    TAG: 'MISSING_TAG'
```

Comment:
is saasherder able to work with these ?

Reply:
I am not aware of any. My understanding was, these variables will be defined in the job while running and then it will be consumed by success_comment at the end of the job.
I want to run this per PR only, not after master-merge.
In this case, do I need to still worry about saasherder support?

Reply:
no, you dont need to worry about saasherder. Thanks for clarifications. I think this is good to merge.